### PR TITLE
Drinks

### PR DESCRIPTION
**Pull request by Fbada006**
This is the drinks part of the app.

### CODE REVIEW

File: app/src/main/java/com/team19/animation/drinks/DrinkCellar.kt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        drinkList.add(Drink("Jameson", 2000, drinkRating = 3f))
+        drinkList.add(Drink("Ciroc", 3000, drinkRating = 4.5f))
+        drinkList.add(Drink("Agua de Valencia", 1500, drinkRating = 3.5f))
+        drinkList.add(Drink("Ruby Dutchess", 12000, drinkRating = 3f))
+        drinkList.add(Drink("Spritzer", 5000, drinkRating = 3f))
+        drinkList.add(Drink("Rourou", 2500, drinkRating = 3f))
+        drinkList.add(Drink("Chocolate Martini", 2500, drinkRating = 3f))
+        drinkList.add(Drink("Apple-Kneel", 2800, drinkRating = 3f))
+        drinkList.add(Drink("Balozi", 3200, drinkRating = 1.5f))
+        drinkList.add(Drink("Kenya Kane", 2000))
```
Comment (keronei) [inline app/src/main/java/com/team19/animation/drinks/DrinkCellar.kt]: Could it be possible the missing rating might result in ```NullPointerException``` ? Please check if there's necessity to add it.

**Comment by Fbada006** [comment]
Hello @keronei. No, you won't get a NPE because I have set default values for it. Check out the Drink data class and you will see.

**Comment by keronei** [comment]
Nice,
Can see there’s ‘insurance’ to that.

**Comment by Fbada006** [comment]
Yeah. You can say that

**Pull request closed without merging**